### DISCUSSION TITLE
10 instances of Nvidia Settings in task bar

### DISCUSSION THREAD

**Discussion by MolecularConcept** [seerge/g-helper#671] [General]
didnt want to put up a bug report , as i dont have the appdata logs,  i have 10 instances of Nvidia Settings in my task bar!  just wanted to bring it up, is it worth taking logs over?

**Comment by seerge** [comment]
@MolecularConcept hello, g-helper definetely didn't launch them. Looks more like an issue with nvidia software?

What you can try is to check "stop all apps using GPU when switching to Eco" in extra settings. It may help to kill nvidia processes every time you turn GPU off 

![Screenshot 2023-06-24 174627](https://github.com/seerge/g-helper/assets/5920850/6774bfc5-01ed-453d-aa00-1879e1c23b17)

**Reply by MolecularConcept** [reply]
thank you for the reply, that option was already enabled. looking into task manager i see a bunch of Nvidia containers? and Nvidia Share ?

**Reply by seerge** [reply]
Does killing those containers (processes) removes tray icons ?

**Reply by MolecularConcept** [reply]
it seems to have sorted itslef out. now i might get one extra container, but havent had one today, but no i could no interact with any of them when they were there.
